modules/timeline.xql

```diff
@@ -27,13 +28,23 @@ for $entry in $entries
 let $dateStr := ($entry/atom:updated, $entry/atom:published)[1]
 let $date := xs:dateTime($dateStr/text())
 let $age := current-dateTime() - $date
+let $minutesSince := minutes-from-duration($age)
+let $hoursSince := hours-from-duration($age)
+let $daysSince := days-from-duration($age)
 let $path := $local:WIKI_ROOT ||substring-after($entry/atom:link[@type eq 'blog']/@href, "/db/apps/wiki/data")
 let $title := $entry/atom:title/text()
-let $abstract := subsequence($entry/atom:content//xhtml:div/xhtml:p[1],1,1)
+let $abstract := subsequence($entry//xhtml:p[1])
```

Comment:
I would expect this to raise a static error, since `fn:subsequence` requires 2 or 3 arguments. To ensure you only get one `<p>` (and not every 1st child `<p>`), I'd go with `head($entry//xhtml:p)`.

Reply:
Oh yes! `$entry//xhtml:p[1]` always gets me. It would have to be `($entry//xhtml:p)[1]` to work the way I was expecting.
On the other hand I only got exactly one paragraph when I tried it in eXide (on a 5.4.0-SNAPSHOT).

Reply:
But your suggestion is a) very explicit and easier to understand and b) will definitely do the right thing.

Reply:
OK then. Still, was it working before then? And when did it stop working (i.e. when did the structure of he posts change)?

Reply:
@lguariento the call to subsequence included the second and third parameter (`1` and `1`)

Reply:
Should it worry us that `//xhtml:p[1]` will definitely only ever return one p-element?
Or is that actually spec compliant?

Reply:
still `head($entry//xhtml:p)` is better

Reply:
I don't think the Atom spec puts any constraints on the structure of the HTML bodies of the entries. I can imagine an entry that contains two divs, each with a first paragraph. So `head()` is safest if we want to limit the results to a single item.